crates/extrinsics/src/call.rs

```diff
@@ -290,14 +292,20 @@ impl CallExec {
             None => self.estimate_gas().await?,
         };
         tracing::debug!("calling contract {:?}", self.contract);
-
-        let call = api::tx().contracts().call(
-            self.contract.clone().into(),
-            self.value.denominate_balance(&self.token_metadata)?,
-            gas_limit.into(),
-            self.opts
-                .compact_storage_deposit_limit(&self.token_metadata)?,
-            self.call_data.clone(),
+        let storage_deposit_limit = self
+            .opts
+            .compact_storage_deposit_limit(&self.token_metadata)?;
+
+        let call = subxt::tx::Payload::new(
+            "Contracts",
+            "call",
+            Call {
+                dest: self.contract.clone().into(),
+                value: self.value.denominate_balance(&self.token_metadata)?,
+                gas_limit: gas_limit.into(),
+                storage_deposit_limit,
+                data: self.call_data.clone(),
+            },
```

Comment:
I'd like to see all extrinsic constructors and types grouped together in a module.